cluster/gce/gci/configure.sh

```diff
@@ -421,7 +421,7 @@ function install-kube-binary-config {
     mv "${src_dir}/kubelet" "${KUBE_BIN}"
     mv "${src_dir}/kubectl" "${KUBE_BIN}"
 
-    mv "${KUBE_HOME}/kubernetes/LICENSES" "${KUBE_HOME}"
+    mv "${KUBE_HOME}/kubernetes/LICENSE" "${KUBE_HOME}"
```

Comment:
this does not exist in CI, see https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-alpha-features/1194514172650459137/artifacts/bootstrap-e2e-master/kube-master-installation.log

Reply:
Reverting just this line fixed some jobs and failed others. see test results in #85210

That means that *some* of our build paths produce this file, and *some* do not.

Reverting this PR in https://github.com/kubernetes/kubernetes/pull/85220 until we can resolve this issue consistently.